index.ts

```diff
@@ -0,0 +1,141 @@
+import { FastifyPluginAsync } from "fastify";
+import { Actor, Item, Member, PostHookHandlerType, PreHookHandlerType, UnknownExtra } from "graasp";
+
+interface GraaspS3FileItemOptions {
+  type: string;
+  sizePath: string;
+}
+
+type MemberExtra = {
+  storage: {
+    total: number;
+  };
+};
+
+const DEFAULT_MAX_STORAGE = 1024 * 1024 * 1024 * 15; // 15GB;
```

Comment:
you should also make this a plugin option (maybe `maxMemberStorage`). make it optional and default to some value, maybe like 100MB for now 🙄